composer.json

```
@@ -24,14 +24,14 @@
         ]
     },
     "require": {
-        "php": "^7.1 || ^8.0",
-        "squizlabs/php_codesniffer": "~3.5.3",
-        "slevomat/coding-standard": "^6.0",
+        "php": "^7.4 || ^8.0",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.5 || ^0.7",
-        "phpstan/phpstan": "^0.12.65",
-        "phpstan/phpstan-strict-rules": "^0.12",
-        "phpstan/phpstan-nette": "^0.12",
-        "phpstan/phpstan-mockery": "^0.12"
+        "phpstan/phpstan": "^1.0",
+        "phpstan/phpstan-mockery": "^1.0",
+        "phpstan/phpstan-nette": "^1.0",
+        "phpstan/phpstan-strict-rules": "^1.0",
+        "slevomat/coding-standard": "6.4.*",
+        "squizlabs/php_codesniffer": "~3.5.3"
```

[o] Jestli k tomu není dobrý důvod, přijde mi lepší semverový zápis.

```suggestion
        "squizlabs/php_codesniffer": "^3.5"
```

*(stejně se instaluje 3.5.6 - 3.5.8)*